libs/components/packages/src/schematics/rules/add-polyfills-config.ts

```diff
@@ -0,0 +1,37 @@
+import { Rule } from '@angular-devkit/schematics';
+
+import { updateWorkspace } from '../utility/workspace';
+
+const polyfillsBuilders = [
+  '@angular-devkit/build-angular:browser',
+  '@angular-devkit/build-angular:dev-server',
+  '@angular-devkit/build-angular:karma',
+  '@blackbaud-internal/skyux-angular-builders:browser',
+  '@blackbaud-internal/skyux-angular-builders:dev-server',
```

Comment:
I don't believe the `dev-server` builders have a `polyfills` option. Are they needed? https://github.com/angular/angular-cli/blob/main/packages/angular_devkit/build_angular/src/builders/dev-server/schema.json

Reply:
You're right. Removed those.